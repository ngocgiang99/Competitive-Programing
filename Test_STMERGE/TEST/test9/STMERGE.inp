7
7 6
14714668 720674266 694896153 682083437 365151405 992404568 
567570732 415568373 252003748 697513120 208956001 713996602 
66272374 7344295 783445051 331866704 841422282 722555554 
710000000 418081677 786357378 359399686 817137974 346084741 
65792966 727879815 252307105 333708557 600503545 431028763 
717110086 91228987 71035329 15807224 124847338 794729315 
37234179 209261713 321737106 657690906 125138403 918326940 
3 10
195594182 805966094 815607619 874781388 593946195 645772316 39336853 757915439 231113325 800119866 
11747200 253108726 643503418 283293215 654133832 906917878 607721502 43251391 332815218 172329027 
656433855 950124366 937785539 459343410 328816897 566498114 938246720 882482008 657594635 77726801 
1 7
173620032 803215982 938376979 183473376 309874245 303705039 102470579 
9 8
415799016 838984979 883583105 675973782 648977520 610100344 588637751 348596844 
814972763 65234791 153075683 969061414 427155165 951531430 431261402 97608105 
975603821 298112824 325835951 395168540 206485438 8950884 504600378 855007970 
430222022 758189 14036621 503323552 267664826 928261594 820408063 906405437 
566774031 267073519 792682276 463692764 459139962 503206193 40123346 488208354 
135423472 360002341 129709085 874125301 429542822 485364625 469198683 16119802 
273930768 635961976 532675871 206050838 980905701 85943841 540610881 318593709 
387698200 174514448 815961043 586204773 565011935 880706570 57117996 382413346 
547403951 208672276 311338977 386957578 979245885 525520499 103984838 7600770 
8 1
116766960 
738324526 
317525927 
707734149 
170854324 
166302196 
331137069 
15251458 
4 7
143962564 471753064 180673953 878496929 587134563 947571516 814889651 
858941074 682437181 553812035 170448435 165020849 23145928 589907842 
847309494 809385506 373986233 643007550 314365202 231443019 944706309 
605363709 806754396 257889488 366853857 629386883 771991527 20111255 
2 4
383902026 44989125 902442589 857714231 
499536474 550689982 76922790 63942704 
