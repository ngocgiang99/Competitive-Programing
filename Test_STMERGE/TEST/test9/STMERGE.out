37234179
89474001
102470579
286086079
15251458
187219747
121911915
